tests/extra/test_job_management.py

```diff
@@ -77,6 +77,8 @@ def sleep_mock():
         yield sleep
 
 
+@pytest.mark.filterwarnings("default:.*`output_file` argument is deprecated.*:DeprecationWarning")
+@pytest.mark.filterwarnings("error")
```

Comment:
Apparently it is also possible to set these markers at module level, (instead of only on this test class), with the `pytestmark` module global var according to https://docs.pytest.org/en/stable/how-to/capture-warnings.html#pytest-mark-filterwarnings